src/Searchbar.vue

```diff
@@ -15,6 +15,10 @@ export default {
   },
   computed: {
     primitiveData() {
+      if (this.value.length < 2) {
+        return undefined;
```

Comment:
Sorry my advice seems to be wrong, returning `undefined` will cause a very weird bug when the value's length goes from 2 to 1 (see below pic). My bad. :P

It's fine to return `[]` instead.

Please do squash your commits after you revert it back, thanks!

<img src="https://user-images.githubusercontent.com/3168908/41020768-ae388144-6996-11e8-8058-8b4a331ed243.gif" width=350 />